flake.nix

```diff
@@ -0,0 +1,39 @@
+{
+  description = "Ratatui - A Rust crate for cooking up Terminal User Interfaces";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    rust-overlay.url = "github:oxalica/rust-overlay";
```

Comment:
Add a comment about why this overlay is necessary. What problem does this solve? Why not fenix or the mozilla overlay?